ui/core/src/utils/text.ts

```diff
@@ -23,3 +27,10 @@ export function dashboardExtendedDisplayName(dashboard: DashboardResource) {
   }
   return dashboard.metadata.name;
 }
+
+export function variableExtendedDisplayName(variable: VariableResource) {
```

Comment:
Could you add a quick inline comment for future devs, I’m a bit confused with how extended display name vs display name is used

Reply:
Sure :)